template/extras/src/ts/middlewares/error-handlers/with-drizzle.ts

```
@@ -0,0 +1,33 @@
+const errorHandler: ErrorRequestHandler = async (
+  err: Error | HttpException,
+  req: Request,
+  res: Response,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  _: NextFunction,
+) => {
+  if (err instanceof HttpException && err.errorCode) {
+    return res.status(err.errorCode).json({ message: err.message, status: err.errorCode });
+  }
+
+  const IP = (req?.headers && req?.headers['x-forwarded-for']) || req.ip;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)